apps/frontend/src/libs/modules/http/libs/enums/enums.ts

```diff
@@ -0,0 +1 @@
+export { HTTPCode, HTTPHeader } from "shared/build/index.js";
```

Comment:
```suggestion
export { HTTPCode, HTTPHeader } from "shared";
```
add `main` and `types` to shared folder

https://github.com/what1s1ove/whatislove.dev/blob/main/packages/shared/package.json#L6